### DISCUSSION TITLE
Micro-tuning Support

### DISCUSSION THREAD

**Discussion by Emad-Marji** [melanchall/drywetmidi#303] [Q&A] [answered]
I am trying to build a tool to support micro-tuning, especially for Turkish and Indian music where non-standard scales are used. I build something based on changing the PitchBend values for each note, but it is not practical. Any idea how to build this?

**Comment by melanchall** [comment]
My first idea is to use pitch bends.

> but it is not practical

So what is practical? Maybe there are some means within the MIDI spec for microtuning?

Please describe your task in more details. Do you want to set a note's pitch finer than one semitone, right?

**Reply by Emad-Marji** [reply]
Correct dear. I am referring to amounts finer than a semitone. This is particularly used in non-Western music where 1/4 or even 1/8 tone is used.
Pitch bend affects all notes on the same MIDI channel equally, so for different micro-tuning adjustments for different notes, I need to use separate MIDI channels. Also, it can be used for mono sounds only, not poly.

**Reply by melanchall** [reply, marked as answer]
I found there is the **MIDI Tuning Standard** which can help you. But I don't know how many synths support this standard. But it allows to tune each note separately as far as I understand. Can you please try to use it?

**Reply by Emad-Marji** [reply]
Very few systems support MTS, making SysEx the best solution available.
